### ISSUE TITLE
Error during install operation

### ISSUE THREAD

**Issue by pureexe** [Microsoft/CNTK#3051]
Following the step from https://docs.microsoft.com/en-us/cognitive-toolkit/Setup-Windows-Binary-Script 

I have download file form https://cntk.ai/dlwg-2.5.html and then I got the error.

```
c:\local\CNTK-2-5\cntk\Scripts\install\windows>install.bat

CNTK Binary Install Script


This script will setup CNTK, the CNTK prerequisites and the CNTK Python environment onto the system.
More help can be found at:
  https://docs.microsoft.com/en-us/cognitive-toolkit/Setup-Windows-Binary-Script

The script will analyse your machine and will determine which components are required.
The required components will be downloaded in [C:\local\CNTK-2-5\cntk\Scripts\install\windows\ps\InstallCache]
Repeated operation of this script will reuse already downloaded components.

 - If required VS2015 Runtime will be installed
 - If required MSMPI will be installed
 - Anaconda3 will be installed into [C:\local\Anaconda3-4.1.1-Windows-x86_64]
 - A CNTK-PY35 environment will be created or updated in [C:\local\Anaconda3-4.1.1-Windows-x86_64\envs]
 - CNTK will be installed or updated in the CNTK-PY35 environment


1 - I agree and want to continue
Q - Quit the installation process

1
Determining Operations to perform. This will take a moment...

The following operations will be performed:
 * Setup CNTK PythonEnvironment 35
 * Setup/Update CNTK Wheel 35
 * Create CNTKPY batch file

Do you want to continue? (y/n)
y
Performing download operations
Download operations finished

Performing install operations
Setup/Update with wheel: https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl. Please be patient....
Collecting cntk==2.5 from https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl
  HTTP error 404 while getting https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl
  Could not install requirement cntk==2.5 from https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl because of error 404 Client Error: Not Found for url: https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl
Could not install requirement cntk==2.5 from https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl because of HTTP error 404 Client Error: Not Found for url: https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl for URL https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl
You are using pip version 8.1.2, however version 9.0.2 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

Fatal error during script execution!
 System.Management.Automation.RuntimeException: Running 'pip install https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl' failed with exit code 1

Error during install operation
```

look like https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp35-cp35m-win_amd64.whl isn't found on server. how can i solve this problem?

best regard
- pakkapon

**Comment by woskii** [comment]
Can you tell me how to download protobuf using script? My exception is like this:
```
Do you want to continue? (y/n)
y
Performing download operations
File [c:\installCacheCntk\cudnn-9.0-windows10-x64-v7.zip] already exists
File [c:\installCacheCntk\cub-1.7.4.zip] already exists
File [c:\installCacheCntk\cmake-3.6.2-win64-x64.msi] already exists
File [c:\installCacheCntk\boost_1_60_0-msvc-14.0-64.exe] already exists
File [c:\installCacheCntk\mklml-mkldnn-0.12.zip] already exists
File [c:\installCacheCntk\swigwin-3.0.10.zip] already exists
Downloading [https://github.com/google/protobuf/archive/v3.1.0.zip], please be patient....
Exception caught - function main / failure
System.Management.Automation.RuntimeException: Download https://github.com/google/protobuf/archive/v3.1.0.zip Failed! WebRequest reported errorCode: []
```

**Comment by pureexe** [comment]
@woskii as same as you, just follow installer script, but it doesn't error on protobuf.

**Comment by Lechtr** [comment]
I have the same problem as pureexe, but with -PyVersion 36 and https://cntk.ai/PythonWheel/GPU/cntk-2.5-cp36-cp36m-win_amd64.whl

**Comment by thiagocrepaldi** [comment]
Fix is under code review already

**Comment by thiagocrepaldi** [comment]
Fixed in a5e0912a2
